tests/run-tests

```diff
@@ -100,35 +96,67 @@ if [ ! -x "$prog_tick_cluster" ]; then
     exit 1
 fi
 
-# Run tick cluster
-_print "Launching tick-cluster..."
-$prog_tick_cluster --interface=127.0.0.1 $package_dir/node_modules/ringpop/main.js >/dev/null &
-pid_tickcluster=$!
-
-sleep 1
-if ! ps -p $pid_tickcluster &>/dev/null; then
-    echo "ERROR: Launching tick-cluster failed" >&2
-    exit 1
-fi
-
 # Clean up tick-cluster on exit
 trap _cleanup exit
 
-# Wait for convergence
-_print "Waiting for convergence..."
-sleep 5
-
 # Test setup
 ${0%/*}/npm-bin-link ../package.json >/dev/null || exit 1
+
 export PATH=$(_normalise_path "${0%/*}/../.bin"):$PATH
+export PROG_RINGPOP="$prog_ringpop"
+export PACKAGE_DIR="$package_dir"
+
+run() {
+	# Check test files exist
+	if ! ls $@ >/dev/null; then
+		echo "ERROR: Test files missing." >&2
+		exit 1
+	fi
+
+	local exit_code=0
+	# Run tick cluster
+	_print "Launching tick-cluster..."
+	$prog_tick_cluster --interface=127.0.0.1 --port 3000 $prog_ringpop >/dev/null &
+	pid_tickcluster=$!
+
+	sleep 1
+	if ! ps -p $pid_tickcluster &>/dev/null; then
+		echo "ERROR: Launching tick-cluster failed" >&2
+		exit 1
+	fi
+
+	# Wait for convergence
+	_print "Waiting for convergence..."
+	sleep 2
```

Comment:
Was on accident to speed up testing on my local machine ;). Will change it back :)